dwave/cloud/api/client.py

```diff
@@ -293,11 +275,8 @@ class DWaveAPIClient:
     # client instance config, populated on init from kwargs overridding DEFAULTS
     config = None
 
-    # User-Agent string used in API requests, as returned by
-    # :meth:`~dwave.cloud.utils.user_agent`, computed on first access and
-    # cached for the lifespan of the class.
     # TODO: consider exposing "user_agent" config parameter
-    user_agent = LazyUserAgentClassProperty()
+    user_agent = default_user_agent()
```

Comment:
Clarifying question, you have a note later where you said you made `default_user_agent()` a function because it might change during runtime. But here you bind it on import to the class, so it won't change later. Does this get overriden somewhere else on instantiation or something like that?

Reply:
good catch, I'll rather initialize it on (session) init.